tutorials/intro/plot_projectors_intro.py

```diff
@@ -0,0 +1,392 @@
+# see the projectors in the output of :func:`~mne.io.read_raw_fif` below:
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file)
+raw.crop(tmax=60).load_data()
+
+###############################################################################
+# In MNE-Python, the environmental noise vectors are computed using `principal
+# component analysis`_, usually abbreviated "PCA", which is why the SSP
+# projectors usually have names like "PCA-v1". The projectors are stored in the
+# ``projs`` field of ``raw.info``:
+
+print(raw.info['projs'])
+
+###############################################################################
+# ``raw.info['projs']`` is an ordinary Python :class:`list` of
+# :class:`~mne.Projection` objects, so you can access individual projectors by
+# indexing into it. The :class:`~mne.Projection` object itself is similar to a
+# Python :class:`dict`, so you can use its ``.keys()`` method to see what
+# fields it contains (normally you don't need to access its properties
+# directly, but you can if necessary):
+
+first_projector = raw.info['projs'][0]
+print(first_projector)
+print(first_projector.keys())
+
+###############################################################################
+# The :class:`~mne.io.Raw`, :class:`~mne.Epochs`, and :class:`~mne.Evoked`
+# objects all have a boolean :attr:`~mne.io.Raw.proj` attribute that indicates
+# whether there are any unapplied / inactive projectors stored in the object,
+# and each individual projector also has a boolean ``active`` field:
+
+print(raw.proj)
+print(first_projector['active'])
+
+###############################################################################
+# Visualizing the effect of projectors
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# You can see the effect the projectors are having on the measured signal by
+# comparing plots with and without the projectors applied. By default,
+# ``raw.plot()`` will apply the projectors in the background before plotting
+# (without modifying the :class:`~mne.io.Raw` object); you can control this
+# with the boolean ``proj`` parameter as shown below, or you can turn them on
+# and off interactively with the projectors interface, accessed via the
+# :kbd:`Proj` button in the lower right corner of the plot window. Here we'll
+# look at just the magnetometers, and a 2-second sample from the beginning of
+# the file.
+
+mags = raw.copy().crop(tmax=2).pick_types(meg='mag')
+mags.plot(butterfly=True, proj=False)
+mags.plot(butterfly=True, proj=True)
+
+###############################################################################
+# Additional ways of visualizing projectors are covered in the tutorial
+# :doc:`../preprocessing/plot_artifacts_correction_ssp`.
+#
+#
+# Loading and saving projectors
```

Comment:
there is no section saying that projectors can also be computed if you want?

Reply:
this was added to the first paragraph of the SSP section. The "artifact rejection with SSP" tutorial doesn't (yet) do EOG/ECG artifacts (only the ICA tutorial does), so I've added a TODO to cross-link to that tutorial once I've added those examples to it.